test/functional/api/buffer_spec.lua

```diff
@@ -29,8 +29,14 @@ describe('buffer_* functions', function()
       eq('', curbuf('get_line', 0))
       curbuf('set_line', 0, 'line1')
       eq('line1', curbuf('get_line', 0))
+      eq('', curbuf('get_line', 1))
+      eq('', curbuf('get_line', -2))
+      eq(false, pcall(function() curbuf('set_line', 1) end))
+      eq(false, pcall(function() curbuf('set_line', -2) end))
```

Comment:
This always returns false because set_line requires 3 arguments.

I'm merging this now, so I will fix it.